package.json

```diff
@@ -48,6 +48,6 @@
     "redux": "^3.6.0",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.2.0",
-    "werewolf-brain": "^2.0.0"
+    "werewolf-brain": "lycan-city/werewolf-brain#feature/32/screenplay"
```

Comment:
Yep, that's the task I added [here](https://github.com/lycan-city/werewolf-moderator/pull/46#issue-234236267). I first need ww-brain to be published to npm for me to be able to merge this pr